terraform/terraform_environment/locals.tf

```diff
@@ -37,7 +37,7 @@ locals {
   sirius_api_gateway_endpoint = var.accounts[local.account_name].sirius_api_gateway_endpoint
   sirius_api_gateway_arn      = var.accounts[local.account_name].sirius_api_gateway_arn
 
-  ecs_task_count = 2
+  ecs_task_count = 0
```

Comment:
Could you reset this to 2 before merging so preprod and prod are running for migration testing please.